core/pom.xml

```diff
@@ -139,6 +139,11 @@
       <artifactId>jackson-databind</artifactId>
       <version>2.3.0</version>
     </dependency>
+    <dependency>
+      <groupId>de.tototec</groupId>
+      <artifactId>de.tototec.cmdoption</artifactId>
```

Comment:
This package is for CommandLineParser. If we add additional parameters like [-parameter value], we can its function conveniently.